static/index.html

```diff
@@ -4,45 +4,45 @@
     <meta charset="utf-8">
     <title>Plotly Database Connector</title>
 
-    <link rel="stylesheet" type="text/css" href="static/css/app.global.css"></link>
-    <link rel="stylesheet" type="text/css" href="static/css/Configuration.css"></link>
-    <link rel="stylesheet" type="text/css" href="static/css/ConnectButton.css"></link>
-    <link rel="stylesheet" type="text/css" href="static/css/DialectSelector.css"></link>
-    <link rel="stylesheet" type="text/css" href="static/css/Link.css"></link>
-    <link rel="stylesheet" type="text/css" href="static/css/Logger.css"></link>
-    <link rel="stylesheet" type="text/css" href="static/css/Preview.css"></link>
-    <link rel="stylesheet" type="text/css" href="static/css/Settings.css"></link>
-    <link rel="stylesheet" type="text/css" href="static/css/TableDropdown.css"></link>
-    <link rel="stylesheet" type="text/css" href="static/css/Tabs.css"></link>
-    <link rel="stylesheet" type="text/css" href="static/css/UserConnections.css"></link>
+    <link rel="stylesheet" type="text/css" href="/static/css/app.global.css"></link>
+    <link rel="stylesheet" type="text/css" href="/static/css/Configuration.css"></link>
+    <link rel="stylesheet" type="text/css" href="/static/css/ConnectButton.css"></link>
+    <link rel="stylesheet" type="text/css" href="/static/css/DialectSelector.css"></link>
+    <link rel="stylesheet" type="text/css" href="/static/css/Link.css"></link>
+    <link rel="stylesheet" type="text/css" href="/static/css/Logger.css"></link>
+    <link rel="stylesheet" type="text/css" href="/static/css/Preview.css"></link>
+    <link rel="stylesheet" type="text/css" href="/static/css/Settings.css"></link>
+    <link rel="stylesheet" type="text/css" href="/static/css/TableDropdown.css"></link>
+    <link rel="stylesheet" type="text/css" href="/static/css/Tabs.css"></link>
+    <link rel="stylesheet" type="text/css" href="/static/css/UserConnections.css"></link>
```

Comment:
this won't work for on-prem where the base URL is `etest.plotly.systems/database-connector`

Reply:
Without absolute hrefs, it fails when we are on urls other than base-url, like:
`domain.name/oauth` --> because these links then start looking for static assets at `domain.name/oauth/static/...`

Reply:
a couple of ways we can get around it:
- option 1: use `HashRouter` instead of `BrowserRouter` to avoid pathname URLs (everything will just be on `/`)
- option 2: template the url base pathname into the HTML
- option 3: serve our static resources in a different way

i think option 1 would be the easiest and i don't have any problems with it personally